locales/de_DE.json

```diff
@@ -2335,33 +2335,33 @@
-      "profile-status-in-review": "Ihr Profil wird derzeit überprüft",
+      "profile-status-in-review": "Profil wird derzeit überprüft",
       "profile-status-completed": "Profil vollständig",
-      "profile-status-rejected": "Ihre Profilangaben wurden abgelehnt",
-      "profile-no-completed-info-message": "Sie müssen alle unten angeforderten Angaben machen, bevor Ihre Limits bestätigt werden können. Dies erfolgt automatisch, sobald alle erforderlichen Informationen bereitgestellt wurden und eine <0>Zahlungsmethode hinzugefügt wird</0>. Das Verfahren kann bis zu fünf Arbeitstage dauern.",
-      "profile-pending-info-message": "Wenn Sie die Details bearbeiten, während Ihr Profil überprüft wird, wird der Überprüfungsprozess neu gestartet.",
-      "profile-rejected-error-message": "Ihre Profilangaben wurden abgelehnt. Bitte aktualisieren Sie Ihre Angaben, um es erneut zu versuchen.",
-      "profile-approved-info-message": "Wenn Sie irgendwelche Profildaten bearbeiten, muss Ihr Konto erneut überprüft werden. Ihre Möglichkeit, digitale Vermögenswerte direkt zu kaufen und zu verkaufen, wird eingefroren, bis diese Überprüfung abgeschlossen ist und Ihre Angaben genehmigt wurden.",
+      "profile-status-rejected": "Deine Profilangaben wurden abgelehnt",
+      "profile-no-completed-info-message": "Du musst alle unten angeforderten Angaben angeben, bevor deine Limits bestätigt werden können. Dies erfolgt automatisch, sobald alle erforderlichen Informationen bereitgestellt wurden und eine <0>Zahlungsmethode hinzugefügt wird</0>. Das Verfahren kann bis zu fünf Arbeitstage dauern.",
+      "profile-pending-info-message": "Wenn du die Details bearbeitest, während dein Profil überprüft wird, wird der Überprüfungsprozess neu gestartet.",
+      "profile-rejected-error-message": "Deine Profilangaben wurden abgelehnt. Bitte aktualisiere deine Angaben, um es erneut zu versuchen.",
+      "profile-approved-info-message": "Wenn du irgendwelche Profildaten bearbeitest, muss dein Konto erneut überprüft werden. Die Möglichkeit, digitale Vermögenswerte direkt zu kaufen und zu verkaufen, wird ausgesetzt, bis diese Überprüfung abgeschlossen ist und deine Angaben genehmigt wurden.",
       "limits-requirements": "Anforderungen",
-      "limits-header": "Jährliche Kauflimits",
-      "limits-complete": "Nash-Plattform Fiat Tier 1",
-      "limits-not-complete": "Deklarieren Sie Beruf und Herkunft der Gelder",
-      "limits-no-occupation": "Besetzung deklarieren",
-      "limits-no-source-of-funds": "Quelle der Mittel deklarieren",
-      "limits-no-level-1": "Muss Tier-1 verifiziert sein",
-      "add-occupation": "Besetzung deklarieren",
-      "add-source-of-funds": "Quelle der Mittel deklarieren",
-      "header": "Tier-1-Profil-Informationen",
+      "limits-header": "Jährliches Kauflimit",
+      "limits-complete": "Nash-Plattform Fiat Stufe 1",
+      "limits-not-complete": "Erkläre deine Tätigkeit und die Herkunft deiner Assets",
```

Comment:
Ist `Beruf` vllt nicht besser als `Tätigkeit`?

`source of funds` -- hier geht's um das Geld, womit man neue Kryptowährungen über Nash kauft. `Herkunft deiner Assets` klingt komisch.

Reply:
beruf vs tätigkeit... ja geht beides

source of funds: klingt sperrig, aber gibt es inhaltlich wieder, aber ist sperrig... vorschläge?